### PR TITLE
Added project FrwTwemoji

### PR DESCRIPTION
**Pull request by FrenchW**
Hi, 
FrwTwemoji is a development around some twemoji components to make it easy to use in any .net project. Full reference is respected
Please, take a look and allow this addiction to the comunity links

### CODE REVIEW

**Comment by WebReflection** [comment]
please note V2 is out ;-)

**Comment by FrenchW** [comment]
Yes, i noticed :-)
It will tale me awake a few nights but I found that the downloaded Unicode reference file have not changed. Where do new numbers come from ?

**Comment by WebReflection** [comment]
that's used only to validate few emoji. In order to avoid possible network errors we create the RegExp parsing available assets. That grants only operation that will succeed will be performed and no surprises with new, unhandled, emoji could possible happen.

Does it make sense?

**Comment by WebReflection** [comment]
TL;DR we parse this folder and have a pretty decent naming convention for files:
https://github.com/twitter/twemoji/tree/gh-pages/2/assets
